MultiSlider.js

```diff
@@ -45,6 +45,7 @@ export default class MultiSlider extends React.Component {
     snapped: false,
     vertical: false,
     minMarkerOverlapDistance: 0,
+    testID: ''
```

Comment:
testID maps to a nullable string so should be able to use `undefined` as a better default value

Reply:
Agreed

Reply:
I followed what was on current master to avoid down-the-line conflicts: https://github.com/ptomasroos/react-native-multi-slider/blob/master/MultiSlider.js#L56

but yeah agreed, will change this

Reply:
updated